lib/config.go

```diff
@@ -131,6 +131,12 @@ type RuntimeConfig struct {
 	// container runtime for all containers.
 	DefaultWorkloadTrust string `toml:"default_workload_trust"`
 
+	// Runtimes defines a list of OCI compatible runtimes. The runtime to
+	// use is picked based on the runtime_handler provided by the CRI. If
+	// no runtime_handler is provided, the runtime will be picked based on
+	// the level of trust of the workload.
```

Comment:
Do we still want to keep the default workload trust level? I though we could now get rid of it since kube dictates about the runtime to use. I don't have a strong opinion though. The linked issue also suggests to deprecate the trust level

Reply:
I know the issue suggests to deprecate the trust level, but what user of k8s are still using the old format to describe their pod? I guess I still want to make sure we won't break use case for those users.
@egernst WDYT?

Reply:
we can deprecate the feature, keep it for 2+ releases or so and get rid of it

Reply:
Yeah I like that. Let's just deprecate it, and when it becomes __hopefully__ widely used (after two more releases as you suggested), we will remove it :)